src/Logger/AuthnRequestIdLoggerDecorator.php

```diff
@@ -22,9 +22,9 @@
 use Surfnet\GsspBundle\Service\StateHandlerInterface;
 
 /**
- * Add's stepup request id to the logs.
+ * Add auth request id to the logs.
  */
-final class StepupRequestIdLoggerDecorator extends AbstractLogger
+final class AuthnRequestIdLoggerDecorator extends AbstractLogger
```

Comment:
Almost there! This logger logs sari, not the request ID of the SAML AuthnRequest.

    $this->stateHandler->getStepupRequestId()

vs

    $this->stateHandler->getRequestId()


    $this->stateHandler->getStepupRequestId()

Reply:
Fixed this